include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,508 @@
+#ifndef _NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_
+#define _NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_
+
+namespace nbl::video
+{
+
+struct SPhysicalDeviceFeatures
+{
+    /* Vulkan 1.0 Core  */
+    bool robustBufferAccess = false;
+    bool fullDrawIndexUint32 = false;
+    bool imageCubeArray = false;
+    bool independentBlend = false;
+    bool geometryShader    = false;
+    bool tessellationShader = false;
+    //VkBool32    sampleRateShading;
+    //VkBool32    dualSrcBlend;
+    bool logicOp = false;
+    bool multiDrawIndirect = false;
+    bool drawIndirectFirstInstance = false;
+    bool depthClamp = false;
+    bool depthBiasClamp = false;
+    bool fillModeNonSolid = false;
+    bool depthBounds = false;
+    bool wideLines = false;
+    bool largePoints = false;
+    bool alphaToOne = false;
+    bool multiViewport = false;
```

Comment:
core in GL 4.1

`IsGLES ? (OES_viewport_array || NV_viewport_array2 || NV_viewport_array):true`